src/app-router.js

```diff
@@ -0,0 +1,43 @@
+import { Routes, Route } from "react-router-dom";
+import LoginPage from "./pages/login";
```

Comment:
As you are creating the `index.ts` file in each page's subfolder, from there you are importing here the pages. So, you can see that you have written 6 or 7 import statements,

What you can do is create an `index.ts` file at the pages folder level (level of the subfolders) and then re-export all the pages from there. And then you would have to write only one import statement here.